package.json

```diff
@@ -15,8 +15,9 @@
     "build": "npm run compile-sass && tsc && npm run copy-views",
     "start": "node $NODE_OPTIONS dist/server.js | bunyan -o short",
     "start:dev": "concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\" \"npm run watch-sass\"",
-    "start-feature": "export $(cat feature.env) && node $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
-    "watch-node-feature": "export $(cat feature.env) && nodemon --watch dist/ $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
+    "oidc-wiremock": "curl --silent --output /dev/null -X POST --data \"@./integration_tests/mockApis/mappings/openid-configuration.json\" http://localhost:9091/__admin/mappings",
+    "start-feature": "npm run oidc-wiremock && export GOVUK_ONE_LOGIN_PRIVATE_KEY=$(cat integration_tests/testKeys/client_private_key.pem) && export $(cat feature.env) && node $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
```

Comment:
Before starting the app, need to ensure the OIDC discovery endpoint is mocked. Also need to make sure the private key is in the environment; can't add it to `feature.env` as it didn't work because of the way this is imported.